docs/advanced/advanced_tech_info.md

```diff
@@ -0,0 +1,617 @@
+
+4.  Display the existing PVC and the created persistent volume \(PV\).
+
+    ```screen
+    $> kubectl get pv,pvc
+    NAME                                                        CAPACITY   ACCESS MODES
+    persistentvolume/pvc-828ce909-6eb2-11ea-abc8-005056a49b44   1Gi        RWO
+    
+    RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS   REASON   AGE
+    Delete           Bound    default/demo-pvc-raw-block   demo-storageclass   109m
+    
+    NAME                             STATUS   VOLUME                                     CAPACITY   
+    persistentvolumeclaim/demo-pvc-raw-block   Bound    pvc-828ce909-6eb2-11ea-abc8-005056a49b44   1Gi
+    
+    ACCESS MODES   STORAGECLASS       AGE
+    RWO            demo-storageclass  78s
+    
+    kubectl describe persistentvolume/pvc-828ce909-6eb2-11ea-abc8-005056a49b44
+    Name:            pvc-828ce909-6eb2-11ea-abc8-005056a49b44
+    Labels:          <none\>
+    Annotations:     pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+    Finalizers:      \[kubernetes.io/pv-protection external-attacher/block-csi-ibm-com\]
+    StorageClass:    demo-storageclass
+    Status:          Bound
+    Claim:           default/demo-pvc-raw-block
+    Reclaim Policy:  Delete
+    Access Modes:    RWO
+    VolumeMode:      Block
+    Capacity:        1Gi
+    Node Affinity:   <none\>
+    Message:
+    Source:
+        Type:              CSI \(a Container Storage Interface \(CSI\) volume source\)
+        Driver:            block.csi.ibm.com
+        VolumeHandle:      SVC:60050760718106998000000000000543
+        ReadOnly:          false
+        VolumeAttributes:      array\_address=baremetal10-cluster.xiv.ibm.com
+                               pool\_name=demo-pool
+                               storage.kubernetes.io/csiProvisionerIdentity=1585146948772-8081-block.csi.ibm.com
+                               storage\_type=SVC
+                               volume\_name=demoPVC-828ce909-6eb2-11ea-abc8-005056a49b44
+    Events:                <none\>
+    
+    ```
+
+5.  Create a StatefulSet.
+
+    ```
+    kubectl create -f demo-statefulset-raw-block.yaml
+    statefulset.apps/demo-statefulset-raw-block created
+    ```
+
+    ```screen
+    $> cat demo-statefulset-raw-block.yaml
+    
+    kind: StatefulSet
+    apiVersion: apps/v1
+    metadata:
+      name: demo-statefulset-raw-block
+    spec:
+      selector:
+        matchLabels:
+          app: demo-statefulset
+      serviceName: demo-statefulset
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: demo-statefulset
+        spec:
+          containers:
+          - name: demo-container
+            image: registry.access.redhat.com/ubi8/ubi:latest
+            command: \[ "/bin/sh", "-c", "--" \]
+            args: \[ "while true; do sleep 30; done;" \]
+            **volumeDevices:
```

Comment:
I think there is a formatting issue with the `**` here

Reply:
thanks. updated

Reply:
same for the filesystem section (this fixed one is in the raw block example)

Reply:
updating now